composer.json

```diff
@@ -50,7 +56,7 @@
         "yiisoft/cache": "^3.0@dev",
         "yiisoft/yii-captcha": "^3.0@dev",
```

Comment:
Seems that this dependency isn't needed.

Reply:
done.